exporters/metric/dogstatsd/dogstatsd.go

```diff
@@ -99,6 +100,37 @@ func NewExportPipeline(config Config, period time.Duration, opts ...push.Option)
 	return pusher, nil
 }
 
+// TODO this could be removed and either tucked into NewExportPipeline if Config is updated or even pushed into invservice
+func NewExportPipelineWithDistributions(config Config, period time.Duration, opts ...push.Option) (*push.Controller, error) {
+	exporter, err := NewRawExporter(config)
+	if err != nil {
+		return nil, err
+	}
+
+	selector := &DistributionSelector{}
+
+	batcher := ungrouped.New(selector, false)
+
+	pusher := push.New(batcher, exporter, period, opts...)
+	pusher.Start()
+
+	return pusher, nil
+}
+
+// DistributionSelector replaces the stock selector to use a DataDog distribution instead of a histogram
+type DistributionSelector struct {}
+
+func (*DistributionSelector) AggregatorFor(descriptor *metric.Descriptor) export.Aggregator {
+	switch descriptor.MetricKind() {
+	case metric.ObserverKind:
+		return lastvalue.New()
+	case metric.MeasureKind:
+		return distribution.New()
```

Comment:
there is not a lot of leverage here...  to determine which instruments should be distributions and/or histograms.